opendebates/views.py

```diff
@@ -352,6 +352,16 @@ def questions(request):
     return redirect(url + "#created=%s" % idea.id)
 
 
+@rendered_with("opendebates/changelog.html")
+def changelog(request):
+    moderated = Submission.objects.filter(
+        Q(approved=False) | Q(duplicate_of__isnull=False)
```

Comment:
No, this isn't a situation where `.distinct()` should be necessary.  The SQL emitted should be something like

    ... WHERE ("opendebates_submission"."approved" = False
               OR "opendebates_submission"."duplicate_of_id" IS NOT NULL)

No opportunity for duplicate rows there.